vtex-apps/shopper-location
# [LOCATION-13] Add region sellers selection component

**What problem is this solving?**
Add new sellers dropdown component. Given the user's location, this component will get all white label sellers serving that region and users will then be able to select one or more sellers. After selection, once the user clicks "Apply" the session will be get updated, removing the `regionId` and adding `private-seller` search facets for the selected sellers instead. This setting will filter the search results to only display products from selected sellers.

Important note, for this feature to work properly, the item cannot have inventory in seller "1". If there's stock in seller "1", then the item will always appear in the search results, regardless of inventory in whichever sellers are selected.

**How should this be manually tested?**
Test in workspace [annasellers](https://annasellers--eriksbikeshop.myvtex.com) in account eriksbikeshop.

1. Set your location to somewhere in the USA
2. Search for the term "Specialized Used Hotrock 16 Kids Bike With Training Wheels"
3. In the dropdown component next to the location displayed in the header, you can select one or more sellers from the region. <br>Selecting _eriksbikeshopburnsville_ should display the "No products were found" page. Selecting 
_eriksbikeshopcoonrapids_ should display SKU 8050751 in the search results. This item only has stock available in the seller _eriksbikeshopcoonrapids_.

**Screenshots or example usage:**
postal code 55306, seller _eriksbikeshopburnsville_
<img width="1440" alt="Screen Shot 2022-08-15 at 5 53 13 PM" src="https://user-images.githubusercontent.com/53097865/184724819-e35541da-6132-477e-a84b-353cc520949c.png">

postal code 55306, seller _eriksbikeshopcoonrapids_
<img width="1440" alt="Screen Shot 2022-08-15 at 5 51 19 PM" src="https://user-images.githubusercontent.com/53097865/184724673-bb620ef2-00ca-4787-af7e-ad30e8bc6bcb.png">
